frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -617,6 +618,42 @@ export const experimentLogic = kea<experimentLogicType>([
                 return !!experiment?.start_date
             },
         ],
+        statusTag: [
```

Comment:
I would refactor this to be:
```
status: [
    (s) => [s.experiment, s.isExperimentRunning],
    (experiment, isExperimentRunning): string =>
        !isExperimentRunning ? 'draft' : !experiment?.end_date ? 'running' : 'complete'
]
```

And then in the component:
```
const logic = experimentLogic({ experimentId: id })
const { status, ... } = useValues(logic)

...

<Tag style={{ alignSelf: 'center' }} color={statusColors[status()]}>
    <b className="uppercase">{status}</b>
</Tag>
```

Reply:
Actually, idk, I like it this way, because it makes sure the same component is used everywhere.

I could instead turn the component into it's own thing, and then use that in both places, rather than the selector itself creating the component.